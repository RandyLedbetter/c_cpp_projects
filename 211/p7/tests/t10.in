insert Chico Sacramento 90
distance Chico Sacramento
