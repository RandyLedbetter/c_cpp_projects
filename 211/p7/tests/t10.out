Chico to Sacramento is 90 miles
